content/code/testing.md

```diff
@@ -66,6 +107,11 @@ Here is an example of such a value being generated.
 
     While randomly generating valid inputs makes sense, randomly generating invalid inputs does not. In the case of invalid inputs, it is much better to have an exhaustive list of all types of cases that are expected to be invalid and always test each one of them.
 
+#### Generic Values Package
```

Comment:
The only types defined in this package are for mocks, so that's what the other section is about. When it comes to generic values, we do not need to expose new types since we're just trying to generate values for the existing types that need to be mocked. Does that make sense?

Reply:
Yes, okay